products/template/tests/test_build.py

```diff
@@ -15,11 +22,11 @@ def test_load_db_tables():
         "dof_shoreline",
         "dpr_greenthumb",
         "lpc_historic_districts",
-        "lpc_landmarks",
+        "nyc_landmarks",
     ]
-    actual_source_tables = get_schema_tables(BUILD_SCHEMA)
-
-    assert actual_source_tables == expected_source_tables
+    actual_tables = pg_client.get_schema_tables(BUILD_SCHEMA)
+    for table in expected_source_tables:
+        assert table in actual_tables
```

Comment:
Any reason not to just throw them in a set and assert on equality?

Reply:
after a full build, there are more tables than just source tables in the schema.

could do other set operations but I'm not sure that the logs of a failed test would list the actual and expected values. when I did something like `assert [e in actual for e in expected]` the pytest logs show something like `<Iterable ... >` instead of the actual values